### DISCUSSION TITLE
Is there a reliable non-laggy way to save progress?

### DISCUSSION THREAD

**Discussion by minotaa** [vinothpandian/react-sketch-canvas#34] [Q&A] [answered]
When you reload the page, your canvas resets, is there a way to save your progress on your canvas without it being excessively laggy on larger projects?

**Comment by vinothpandian** [comment]
You can get the paths using onUpdate method, store it in local or session storage and load it back using loadPaths method

**Reply by minotaa** [reply]
I tried this prior and it causes immense lag on Chrome. Additionally, it causes a lot of errors relating to the quota.

**Reply by minotaa** [reply]
Still need help on this :(

**Reply by vinothpandian** [reply, marked as answer]
Can you use the underlying canvas and stream the information to the server instead? Check #14 for example usage.
